<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bgWidget</class>
 <widget class="QDialog" name="bgWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#bgWidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0.369, x2:1, y2:0.756, stop:0 rgba(237, 66, 100, 255), stop:1 rgba(255, 237, 188, 255));
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>181</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;;
color:rgb(65, 67, 106);</string>
   </property>
   <property name="text">
    <string>welcome</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mode_1">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>170</y>
     <width>251</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(57, 174, 169);</string>
   </property>
   <property name="text">
    <string>child is alone at home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="member">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>110</y>
     <width>211</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(57, 174, 169);</string>
   </property>
   <property name="text">
    <string>add new member</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mode_2">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>230</y>
     <width>251</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(57, 174, 169);</string>
   </property>
   <property name="text">
    <string>no one is home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mode_3">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>290</y>
     <width>251</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(57, 174, 169);</string>
   </property>
   <property name="text">
    <string>automatic mode</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
